CHANGELOG.md

```diff
@@ -19,3 +20,7 @@ should be replaced by:
     // Restrict the length to a value less than 5
     let len: usize = kani::any_where(|x| *x < 5);
 ```
+
+### Major Changes
+
+- Enable the build cache by default and introduce `--force-build` to build from scratch by @celinval in #2232. 
```

Comment:
Can you elaborate a bit on the impact of this change?

Reply:
How verbose do we want the changelog to be?

Reply:
Doesn't need to be too verbose. We can instead edit the PR description a bit to show how this impacts the user, e.g. "Enable incremental compilation to avoid recompiling crates that haven't changed".

Reply:
Incremental compilation is something different in the rust context. But I can definitely add the user impact. Thanks